packages/console/src/pages/SignInExperience/utils/constants.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  type LocalePhraseGroupKey,
+  type LocalePhraseKey,
+} from '@logto/phrases-experience/lib/types';
+import { conditionalArray } from '@silverhand/essentials';
+
+import { isDevFeaturesEnabled } from '@/consts/env';
+
+export const hiddenLocalPhraseGroups: readonly LocalePhraseGroupKey[] = [
+  'demo_app',
+  ...conditionalArray(!isDevFeaturesEnabled && 'mfa'),
+];
+
+export const hiddenLocalPhrases: readonly LocalePhraseKey[] = [
+  ...conditionalArray(
+    !isDevFeaturesEnabled &&
+      ([
+        'action.copy',
+        'action.verify_via_passkey',
+        'action.download',
+        'input.backup_code',
```

Comment:
I think we don't need to hide `action.copy` and `action.download`

Reply:
Place `verify_via_passkey` and `backup_code` in `mfa` group? This way we don't have to hide them seperately

Reply:
the hidden feature is intended for convenient feature control during the development process. Therefore, we will retain the current implementation.